server/repository/src/db_diesel/invoice.rs

```diff
@@ -234,6 +237,14 @@ fn create_filtered_query<'a>(filter: Option<InvoiceFilter>) -> BoxedInvoiceQuery
         apply_date_time_filter!(query, shipped_datetime, invoice_dsl::shipped_datetime);
         apply_date_time_filter!(query, delivered_datetime, invoice_dsl::delivered_datetime);
         apply_date_time_filter!(query, verified_datetime, invoice_dsl::verified_datetime);
+
+        if let Some(stock_line_id) = stock_line_id {
```

Comment:
nice

Reply:
At some point we will be displaying stock line movements (ledger), this would be very useful there too.